internal/usage/usage.go

```diff
@@ -440,16 +440,16 @@ dbName and collection are required only for built-in roles.`
 	SourceCollectionName                      = "Human-readable label that identifies the source collection."
 	SourceDatabaseName                        = "Human-readable label that identifies the source database."
 	SourcePolicyItemID                        = "Human-readable label that identifies the policy item."
-	Transform                                 = "Fields to be excluded for this data lake pipeline."
+	Transform                                 = "Fields to exclude for this data lake pipeline."
 	PipelineFilename                          = "Name of the JSON data lake pipeline configuration file to use."
 	Pipeline                                  = "Name of the Data lake pipeline."
-	CompletedAfter                            = "Date filter of which the backup snapshots where completed. Format must be YYYY-MM-DD"
+	CompletedAfter                            = "Date filter for when the backup snapshots were completed. You must use the YYYY-MM-DD format."
 	Tag                                       = "List that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the cluster."
 	DeploymentTag                             = "List that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the deployment."
 	ServerlessTag                             = "List that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the serverless instance."
-	UpdateWarning                             = " Passing this flag replaces preexisting data."
-	EnableWatch                               = "Watch the command until it completes its execution or the watch times out. To set the time that the watch times out, use the --watchTimeout option."
-	EnableWatchDefault                        = "Watch the command until it completes its execution or the watch times out."
+	UpdateWarning                             = "Flag that indicates whether passing this flag replaces preexisting data."
```

Comment:
I haven't seen the original copy suggestion, but I think this makes more sense
```suggestion
	UpdateWarning                             = "Flag that indicates whether to replace preexisting data."
```

Reply:
updated! keeping this for history